### PR TITLE
Add Cache console

### PR DESCRIPTION
**Pull request by MrPetovan**
I needed to check the cache content to debug an addon, so I made it a full-featured console command.

### CODE REVIEW

**Comment by nupplaphil** [comment]
Great tool! I recently used redis-cli for debugging. This is a rather a better way :-)

**Comment by MrPetovan** [comment]
I'm not sure how to write test for this yet, I need to study what you did before.

**Comment by nupplaphil** [comment]
A guess f.e.:
- Add manually some keys
- use the getallkeys method and check if you really got all keys

Another idea:
For the config you can replace/enhance some of the existing tests for get/set/add methods of the abstract cache test with the console commands.

Because they will get implicit called with the console command and should work the same way :-)

File: src/Core/Cache/MemcacheCacheDriver.php [hunk 1]
```diff
@@ -35,6 +40,28 @@ public function __construct($memcache_host, $memcache_port)
...
 	}
 
+	/**
+	 * (@inheritdoc)
+	 */
+	public function getAllKeys()
+	{
+		$list = [];
+		$allSlabs = $this->memcache->getExtendedStats('slabs');
+		foreach($allSlabs as $slabs) {
```
Comment (annando) [inline src/Core/Cache/MemcacheCacheDriver.php]: Just some standards here and in the following lines: Could you add a space after "foreach"?

**Comment by annando** [comment]
Can you change the missing spaces? I would like to merge this.

**Pull request merged**